Makefile

```diff
@@ -136,4 +164,25 @@ delete-ccs-accountclaim:
 
 # Test CCS
 .PHONY: test-ccs
-test-ccs: create-ccs-namespace create-ccs-secret create-ccs-accountclaim delete-ccs-accountclaim delete-ccs-secret delete-ccs-namespace
\ No newline at end of file
+test-ccs: create-ccs-namespace create-ccs-secret create-ccs-accountclaim delete-ccs-accountclaim delete-ccs-secret delete-ccs-namespace
+
+# Create S3 bucket
+.PHONY: create-s3-bucket
+create-s3-bucket:
+	# Get credentials
+	@export AWS_ACCESS_KEY_ID=$(shell oc get secret osd-creds-mgmt-osd-staging-1-osdmanagedadminsre-secret -n aws-account-operator -o json | jq -r '.data.aws_access_key_id' | base64 -d); \
+	export AWS_SECRET_ACCESS_KEY=$(shell oc get secret osd-creds-mgmt-osd-staging-1-osdmanagedadminsre-secret -n aws-account-operator -o json | jq -r '.data.aws_secret_access_key' | base64 -d); \
+	aws s3api create-bucket --bucket ${REUSE_BUCKET_NAME} --region=us-east-1
+
+# List S3 bucket
+.PHONY: list-s3-bucket
+list-s3-bucket:
+	# Get credentials
+	export AWS_ACCESS_KEY_ID=$(shell oc get secret osd-creds-mgmt-osd-staging-1-osdmanagedadminsre-secret -n aws-account-operator -o json | jq -r '.data.aws_access_key_id' | base64 -d); \
+	export AWS_SECRET_ACCESS_KEY=$(shell oc get secret osd-creds-mgmt-osd-staging-1-osdmanagedadminsre-secret -n aws-account-operator -o json | jq -r '.data.aws_secret_access_key' | base64 -d); \
```

Comment:
Just a suggestion for getting the secrets without using `jq` or `base64 -d`:

`oc extract secret/osd-creds-mgmt-osd-staging-1-osdmanagedadminsre-secret -n aws-account-operator --keys=aws_secret_access_key --to=- 2>&1 | tail -n 1 `

Just throwing it out there as an option.